components/CreateOrganization.js

```diff
@@ -39,100 +37,130 @@ class CreateOrganization extends React.Component {
+              role: member.role,
+              member: {
+                id: member.member.id,
+                name: member.member.name,
+              },
+            })),
+          },
+        });
+      }
       await this.props.refetchLoggedInUser();
+
       Router.pushRoute('collective', {
-        CollectiveId: collective.id,
-        slug: collective.slug,
+        slug: response.data.createOrganization.slug,
         status: 'collectiveCreated',
-      });
+      }).then(() => window.scrollTo(0, 0));
     } catch (err) {
       const errorMsg = getErrorFromGraphqlException(err).message;
-      this.setState({ result: { error: errorMsg } });
+      this.setState({ result: { error: errorMsg }, status: 'error' });
       throw new Error(errorMsg);
     }
   }
 
   render() {
     const { LoggedInUser } = this.props;
-
-    const title = 'Create a new organization';
+    const { result, collective, status } = this.state;
 
     return (
-      <div className="CreateOrganization">
-        <Header
-          title={title}
-          className={this.state.status}
-          LoggedInUser={LoggedInUser}
-          menuItems={{ pricing: true, howItWorks: true }}
-        />
-
+      <Container>
         <Body>
-          <Container mt={2} mb={2}>
-            <H1 fontSize={['24px', '40px']} lineHeight={3} fontWeight="bold" textAlign="center" color="black.900">
-              {title}
-            </H1>
-            <P textAlign="center">
-              <FormattedMessage
-                id="collectives.create.description"
-                defaultMessage="An Organization allows you to make financial contributions as a company or team. You can also add a credit card with a monthly limit that team members can use to make contributions."
-              />
-            </P>
-          </Container>
-
-          <div className="content">
+          <Container>
             {!LoggedInUser && (
               <Container textAlign="center">
                 <SignInOrJoinFree />
               </Container>
             )}
             {LoggedInUser && (
-              <div>
+              <Container>
                 <CreateOrganizationForm
-                  collective={this.state.collective}
-                  onSubmit={this.createCollective}
+                  collective={collective}
+                  onSubmit={this.createOrganization}
                   onChange={this.resetError}
+                  error={result.error}
+                  updateAdmins={this.updateAdmins}
+                  loading={status === 'loading'}
                 />
-
-                <Container textAlign="center" marginBottom="5rem">
+                <Container
+                  textAlign="center"
+                  alignItems="center"
+                  justifyContent="center"
+                  marginBottom="5rem"
+                  width={[100, 200, 600]}
+                >
                   <Container color="green.500">{this.state.result.success}</Container>
```

Comment:
What is this for? If we have a 'success' it redirects to the new organization page, so I don't think we ever would see this. Is this meant to show the error?

Reply:
Good Catch! Leftover from old refactor not needed anymore.